python/source/impedance_spectroscopy.py

```diff
@@ -1,3 +1,5 @@
+# This Python file uses the following encoding: iso-8859-1
+#
```

Comment:
No it needs to be within the first two lines. See https://www.python.org/dev/peps/pep-0263/